src/test/java/com/puppycrawl/tools/checkstyle/AuditEventDefaultFormatterTest.java

```diff
@@ -104,6 +104,23 @@ public void testFormatModuleNameDoesNotContainCheckSuffix() {
     }
 
     @Test
+    public void testFormatModuleWithModuleId() {
+        final AuditEvent mock = PowerMockito.mock(AuditEvent.class);
```

Comment:
I don't know why this is, but since ever test is like this, we'll just leave it as is.